SAK-30334 Incorporate NYU's "favorites" changes

marktriggs opened the pull request:
Since this work covers multiple areas of Sakai, I've split it into 3 independent commits (and I've tested that they work individually).  I've put some background and screenshots of the changes in the JIRA (https://jira.sakaiproject.org/browse/SAK-30334)

CC'ing @jeffpasch and @kyleblythe to answer any questions about the visual design.

juanjmerono commented:
Could you squash this 3 commits into a single one?

SedueRey commented on reference/library/src/morpheus-master/sass/modules/more-sites/_base.scss:
```diff
@@ -63,6 +109,38 @@ ul#otherSitesMenu{
 	}
 }
 
+$narrowDisplay: "only screen and (max-width : 1200px)";
```
Hi, great work indeed.

Could you add this breakpoint to _defaults.scss with `$break-small-end` and `$break-medium-end` to keep it organized.

marktriggs commented:
@juanjmerono No problem.  I've squashed those now and attached the JIRA id.

@SedueRey Thanks for checking it out!  I've moved that breakpoint declaration into defaults.scss.  I named it after the "more sites" feature (since it's specific to that feature).  Happy to name it to something else if you have a better idea: 
https://github.com/sakaiproject/sakai/commit/21c0baf1846d95cbf5d0d0f8d0b078f97c8e6f59#diff-0018a005228d6217ce40b9adeb3a1e30R19

Cheers,
Mark

SedueRey commented:
LGTM, great name :) ... I liked last name too hehehe. I had no problem on that
